remoting/exchange.go

```diff
@@ -0,0 +1,128 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package remoting
+
+import (
+	"time"
+
+	"github.com/apache/dubbo-go/common"
+	"go.uber.org/atomic"
+)
+
+var (
+	sequence atomic.Uint64
+)
+
+func init() {
+	sequence.Store(0)
+}
+
+func SequenceId() uint64 {
+	return sequence.Add(2)
+}
+
+// Request ...
+type Request struct {
+	Id       int64
+	Version  string
+	SerialID byte
+	Data     interface{}
+	TwoWay   bool
+	Event    bool
+	broken   bool
+}
+
+// NewRequest ...
+func NewRequest(version string) *Request {
+	return &Request{
+		Id:      int64(SequenceId()),
+		Version: version,
+	}
+}
+
+// Response ...
+type Response struct {
+	Id       int64
```

Comment:
in dubbbo-go, the `Id` should be `ID`. and also, we use `URL/HTTP` etc.

Reply:
Fixed